content/community/gsoc/2024/_index.html

```diff
@@ -54,7 +54,7 @@ <h3>Who are the Haiku mentors?</h3>
 would otherwise depend on a single person inside a volunteer driven project.
 This also helps to strengthen the students' ability and comfort with discussions
 on an open mailing list, which is essential in many remote software development
-models. A complete list of all Haiku mentors for <i>Google Summer of Code</i> 2023 is being compiled.</p>
+models. A complete list of all Haiku mentors for <i>Google Summer of Code</i> 2024 is being compiled.</p>
```

Comment:
It is compiled now, so we can update this as well:

- Jérôme Duval (Korli)
- Niels Reedijk (nielx)
- Adrien Destugues (PulkoMandy)
- Rene Gollent (AnEvilYak)
- Augustin Cavalier (waddlesplash)